sycl/include/CL/sycl/kernel_bundle.hpp

```diff
@@ -309,6 +312,9 @@ class kernel_bundle : public detail::kernel_bundle_plain {
     return reinterpret_cast<device_image_iterator>(kernel_bundle_plain::end());
   }
 
+  /// \returns the number of device images in the kernel_bundle
+  size_t size() const { return kernel_bundle_plain::size(); }
```

Comment:
Is this method defined by the specification(cannot find it)?
Suggest accessing kernel_bundle_impl::size directly.

Reply:
Agreed, the spec does not have a `size()` member function, so we shouldn't add one.

Reply:
Fixed